### DISCUSSION TITLE
Is there a way to change default storage?

### DISCUSSION THREAD

**Discussion by w4sb0y** [nextcloud/all-in-one#3517] [Questions] [answered]
I have added an external disk to my raspberry pi, on which nextcloud runs on, and I wonder if I can change where nextcloud saves files by default? It would be much better if it didn't save files onto the default SD card, only the hard disk. Here's a picture of the attached disks:
![image](https://github.com/nextcloud/all-in-one/assets/68382177/8074bfaa-4461-4809-b9e8-ddbaa9f08133)

Thanks for the answer(s) in advance c:

**Comment by szaimen** [comment, marked as answer]
Hi, I guess you are looking for this? https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive

**Reply by w4sb0y** [reply]
Yep, thanks!
